index.html

```diff
@@ -33,247 +33,337 @@
-                <button onclick="userChoice('Rock')"> <img src="https://static.thenounproject.com/png/477914-200.png"
-                        width="200" height="200" alt="An Image of a Rock" /> </button>
-            </div>
-            <h2 id="score" class="middle"></h2>
-            <div id="bChoices">
-                <img src="https://static.thenounproject.com/png/477914-200.png" width="200" height="200"
-                    alt="An Image of a Rock" />
-                <img src="https://static.thenounproject.com/png/477912-200.png" width="200" height="200"
-                    class="botcards" alt="An image of a Paper" />
-                <img src="https://static.thenounproject.com/png/88666-200.png" width="200" height="200" class="botcards"
-                    alt="An image of a Scissor" />
-            </div>
+        <div id="bChoices">
+          <img
+            src="https://static.thenounproject.com/png/477914-200.png"
+            width="200"
+            height="200"
+            alt="An Image of a Rock"
+          />
+          <img
+            src="https://static.thenounproject.com/png/477912-200.png"
+            width="200"
+            height="200"
+            class="botcards"
+            alt="An image of a Paper"
+          />
+          <img
+            src="https://static.thenounproject.com/png/88666-200.png"
+            width="200"
+            height="200"
+            class="botcards"
+            alt="An image of a Scissor"
+          />
         </div>
-        <button id="restart" class="restartHide" onclick="restart()">Restart</button>
-
-
+      </div>
+      <button id="restart" class="restartHide" onclick="restartGame()">
+        Restart
+      </button>
     </div>
     <script>
-        const cards = document.getElementById("sGame");
-        const start = document.getElementById("start");
-        const match = document.getElementById("game");
-        const progress = document.getElementById("score");
-        const nameField = document.getElementById("nameField");
-        const restartButton = document.getElementById("restart");
-        const pCards = document.getElementById("pChoices");
-        const bCards = document.getElementById("bChoices");
-        const gBoard = document.getElementById("board");
+      // Constants
+      const kRock = "Rock";
+      const kScissors = "Scissors";
+      const kPaper = "Paper";
 
-        const startState = "START";
-        const playState = "PLAY";
-        const endState = "END";
+      const kStartState = "Start";
+      const kPlayState = "Play";
+      const kEndState = "End";
 
-        let gameState = startState;
+      const kWin = "Win";
+      const kLose = "Lose";
+      const kTie = "Tie";
 
-        let win = 0;
-        let lose = 0;
+      // Elements
+      const cards = document.getElementById("sGame");
+      const startElement = document.getElementById("start");
+      const match = document.getElementById("game");
```

Comment:
try to name these better. You have `match` but the id is `game`. This will make the code confusing. Also try to add a suffix `element` / `button` to make it more clear that it is an HTML element

Reply:
To go over element/button